smoke-test/src/main/java/org/opennms/smoketest/utils/RestClient.java

```diff
@@ -419,7 +419,7 @@ public JsonNode getBackups() throws IOException {
                 .accept(MediaType.APPLICATION_JSON)
                 .get(String.class);
 
-        if (result == null) {
+        if (result == null || result.isEmpty()) {
```

Comment:
This is necessary, as due to the removal of jasper-reports, now jersey is actually used. Before apache cxf client was used (leaked in via the opennms reporting dependencies). It turns out, jersey and cxf have slightly different behavior.